docs/get-started/using-with-map.md

```diff
@@ -35,9 +40,15 @@ If you are using react-map-gl, there are several ways to provide a token to your
 * Pass it as a prop to the ReactMapGL instance `<ReactMapGL mapboxApiAccessToken={TOKEN} />`
 
 
-### Display Maps Without A Mapbox Token
+### Using other basemap services, or your own
```

Comment:
Let's keep the headers concise:

- Using Mapbox basemap service 
- Using other basemap services
- Hosting your own basemap service